src/Umbraco.Web.Common/Formatters/AngularJsonMediaTypeFormatter.cs

```diff
@@ -21,8 +21,7 @@ public class AngularJsonMediaTypeFormatter : NewtonsoftJsonOutputFormatter
         public AngularJsonMediaTypeFormatter(JsonSerializerSettings serializerSettings, ArrayPool<char> charPool, MvcOptions mvcOptions)
             : base(serializerSettings, charPool, mvcOptions)
         {
-            serializerSettings.Converters.Add(new VersionConverter());
```

Comment:
I assume this change is for your note: 

> Bug fix found by the test of TemplateQueryController.
> Output was not camelcased.
> Fix in AngularJsonOnlyConfigurationAttribute.cs

But... does this now mean everything is now camel cased by default? If so, we cannot have that because angular expects many things to not be auto-camel cased. Also what about the VersionConverter?

Reply:
This change bacially just forces the `serializerSettings` to be injected..

So the `VersionConverter` is still used by `AngularJsonOnlyConfigurationAttribute` and camelcase is only used by `JsonCamelCaseFormatterAttribute`